src/index.ts

```diff
@@ -74,7 +74,8 @@ export function useMappedState<TState, TResult>(
         }
       };
 
-      // Pull data from the store on first render.
+      // Pull data from the store after first render in case the store has
```

Comment:
fwiw neither of these are only after "first" render — they all execute on updates to the selector too